app/views/bulk_reference.scala.html

```diff
@@ -30,7 +30,7 @@
 @layout(
     applicationConfig = ac,
     title =  title,
-    backLink = Some(BackLinkParams(href = controllers.routes.FileUploadController.get.url)),
+    backLink = Some(BackLinkParams(href = "javascript:history.back()")),
```

Comment:
Any JS links we use should have the handling abstracted from the markup and they must respect users without JS - the way to achieve this with this API is to use "#" as the href value, there is handling in Layout where this param is consumed to convert hash links to JS links using abstracted handling in javascripts/back.js and hide the link from noJS users.

Reply:
resolved in [09a0398](https://github.com/hmrc/gmp-frontend/pull/444/commits/09a03984924c929faa709339b444d60abbde44fd)